docker/docker_update.sh

```diff
@@ -1,6 +1,53 @@
 #!/bin/sh
 set -e
 
+Help()
+{
+  # Display help
+  echo "Usage: ./docker_update.sh [OPTIONS]"
+  echo ""
+  echo "Start all docker containers required to run Lemmy."
+  echo ""
+  echo "Options:"
+  echo "-u Docker username. Only required if managing Docker via Docker Desktop with a personal access token."
+  echo "-h Print this help"
+}
+
+while getopts ":hu:" option; do
+  case $option in
+    h) Help
+       exit;;
+    u) DOCKER_USER=$OPTARG
+       ;;
+    *) echo "Invalid option $OPTARG."
+       exit;;
+  esac
+done
+
+# uname may not exist on windows machines; default to unknown to be safe.
+ARCH=$(uname -m 2>/dev/null || echo 'unknown')
+
 mkdir -p volumes/pictrs
+
+echo "[🐀 lemmy] Need user password to run 'sudo chown' for the pictrs volume."
 sudo chown -R 991:991 volumes/pictrs
-sudo docker compose up -d --build
+
+if [ "$ARCH" = 'arm64' ]; then
+  echo "[🐀 lemmy] WARN: If building from images, make sure to uncomment 'platform' in the docker-compose.yml file!"
```

Comment:
This can be removed once https://github.com/LemmyNet/lemmy/pull/2806 is complete.